packages/loader/container-loader/src/container.ts

```diff
@@ -975,7 +980,12 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
             // Note: no need to fetch ops as we do it preemptively as part of DeltaManager.attachOpHandler().
             // If there is gap, we will learn about it once connected, but the gap should be small (if any),
             // assuming that connect() is called quickly after initial container boot.
-            this.connectInternal({ reason: "DocumentConnect", fetchOpsFromStorage: false });
+            this._deltaManager.connectAbortController = new AbortController(); // Reset the signal
```

Comment:
This does not look right. We should not reach out and change state of an object.
Especially that it's not really used by DeltaManager - you can stick it on Container with same result.

But most important - what happens if this code is called twice? Second call will happily bail out from connectCore() because there is pending connection and never touch this abort signal. And as result, calling cancel on it would do nothing.

That's the reason I suggested earlier one of these two flows:
1. connec() returns you back IProgress - i.e. reverse the flow. It would need to create new one (if there is no pending connection, as well as story it for later time), or return existing one
2. Keep existing flow, but daisy-chain abort signals together - one that is stored in DeltaManager from first call, and another one -that is passed in. I had sample in the issue where we discussed it.